pkg/apis/pipeline/v1beta1/pipeline_validation_test.go

```diff
@@ -1162,6 +1162,82 @@ func TestValidatePipelineResults_Failure(t *testing.T) {
 	}
 }
 
+func TestFinallyTaskResultsToPipelineResults_Failure(t *testing.T) {
+	tests := []struct {
+		desc          string
+		p             *Pipeline
+		expectedError apis.FieldError
+		wc            func(context.Context) context.Context
+	}{{
+		desc: "invalid propagation of finally task results from pipeline results",
```

Comment:
could you please simplify this test case? for example I don't think there's any need for workspaces

Reply:
agreed, outside of workspaces I don't believe there is anything else that can be taken out to make the test case simplified.

Reply:
you could do a single pipeline result, no pipeline tasks, and one finally task; also the finally task doesn't need params

Reply:
yes, everything has been update but in order to have a finally field there must be a pipeline task.